data/services/zeppelin.py

```diff
@@ -34,19 +35,25 @@ class ZeppelinServiceOnBI(AbstractServiceOnBI):
     }
 
     def start(self):
-        print "starting zeppelin"
-        stdin, stdout, stderr = self.ssh.exec_command(self.dirName + "/bin/zeppelin-daemon.sh status")
-        zeppelin_status = stdout.read()
-        if "OK" in zeppelin_status:
-            print "Zeppelin already running. Restarting instead"
-            stdin, stdout, stderr = self.ssh.exec_command("zeppelin-0.7.0-SNAPSHOT/bin/zeppelin-daemon.sh restart")
-            print stdout.read()
-        if "FAILED" in zeppelin_status:
-            stdin, stdout, stderr = self.ssh.exec_command(self.dirName + "/bin/zeppelin-daemon.sh start")
+        # Add quick check here to see if s3AuthSet was run
+        if(self.s3authSet) : 
```

Comment:
This introduces API breaking changes.  The example scripts need to be updated from:

```python
z = ZeppelinServiceOnBI(SERVER, USERNAME, PASSWORD)
z.install()
z.start()
sleep(5)
z.downloadConfig({"interpreter.json"  : "conf/interpreter.json"})
z.updateConfig()
#z.setS3auth(S3_USERNAME, S3_BUCKET) ## Must do this first!
```

to 

```python
z = ZeppelinServiceOnBI(SERVER, USERNAME, PASSWORD)
z.install()
z.setS3auth(S3_USERNAME, S3_BUCKET) ## Must do this first!
z.start()
sleep(5)
z.downloadConfig({"interpreter.json"  : "conf/interpreter.json"})
z.updateConfig()
```